examples/Cargo.toml

```
@@ -9,13 +9,30 @@ rust-version = "1.60"
 [package.metadata.release]
 release = false
 
+[features]
+default = ["rustls-tls"]
+rustls-tls = ["kubert/rustls-tls"]
+openssl-tls = ["kubert/openssl-tls", "openssl"]
+
+[dependencies.kubert]
+path = "../kubert"
+default-features = false
+features = ["clap", "lease", "runtime", ]
```

This overlaps with the dev-dependencies in a confusing way... Should we remove the dev dep block?

Also
```suggestion
[dependencies.kubert]
path = "../kubert"
default-features = false
features = ["clap", "lease", "runtime"]
```

---

yeah, i think we can remove the dev dep, my bad!